cloud/drop-databases.yml

```diff
@@ -0,0 +1,13 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - ../concent-builder/consts.yml
+    - "{{ deployment_values }}/var.yml"
+    - "{{ deployment_values }}/var-{{ cluster }}.yml"
+    - "{{ local_secret_dir }}/{{ cluster }}/cluster-secrets.yml"
+    - "{{ local_secret_dir }}/cloud/cloud-secrets.yml"
+  vars:
+    choose_database_option: delete
+  roles:
+    - { role: postgres_connection, cluster_type: control}
+    - { role: postgres_connection, cluster_type: storage}
```

Comment:
Please fix style here.

Reply:
The style is the same as for python ?
```python
{
    'key1': 'value1',
    'key2': 'value2',
}
```

Reply:
No, I meant the whitespace. Your code above looks like this: `{ a: b}`.
It should be either `{ a: b }` or `{a: b}` instead.

Reply:
Done.